src/css/themes/bootstrap3/default/icons-styles.css

```diff
@@ -2,10 +2,15 @@
YHAQYWMyEyNicBLgEjFRchAgAEBwP+7ggKDAIkDAoI/u4DBgUEBwP+7ggKDAIkDAoI/u4DBgWi/rxJBAMBSQkVFQn+twQDA24EA/63CRUVCQFJBANU0QAAAgAAANsEAAMlABoANQAAEyIGFREUFjMhMjY9ATMyNj0BNCYrATU0JiMhFyEyFh0BMzIWHQEUBisBFRQGIyEiJjURNDYzSR4rKx4C3B4rJC5AQC4kKx79JCUCkg8WbQ8WFg9tFg/9bg8WFg8DJSsf/kofKysfSUAtSi1ASR8rShUPbhUPSg8Vbg8VFQ8Bbg8VAAAAAAMAAADbBAADJQAaADUARQAAEyIGFREUFjMhMjY9ATMyNj0BNCYrATU0JiMhFyEyFh0BMzIWHQEUBisBFRQGIyEiJjURNDYzFzMyFh0BFAYrASImPQE0NkkeKyseAtweKyQuQEAuJCse/SQlApIPFm0PFhYPbRYP/W4PFhYPSUkPFhYPSQ8WFgMlKx/+Sh8rKx9JQC1KLUBJHytKFQ9uFQ9KDxVuDxUVDwFuDxVJFQ/cDxUVD9wPFQAEAAAA2wQAAyUAGgA1AEUAVgAAEyIGFREUFjMhMjY9ATMyNj0BNCYrATU0JiMhFyEyFh0BMzIWHQEUBisBFRQGIyEiJjURNDYzFzMyFh0BFAYrASImPQE0NjsBMhYdARQGKwEiJj0BNDYzSR4rKx4C3B4rJC5AQC4kKx79JCUCkg8WbQ8WFg9tFg/9bg8WFg9JSQ8WFg9JDxYW6kkQFRUQSQ8VFQ8DJSsf/kofKysfSUAtSi1ASR8rShUPbhUPSg8Vbg8VFQ8Bbg8VSRUP3A8VFQ/cDxUVD9wPFRUP3A8VAAUAAADbBAADJQAaADUARQBWAGYAABMiBhURFBYzITI2PQEzMjY9ATQmKwE1NCYjIRchMhYdATMyFh0BFAYrARUUBiMhIiY1ETQ2MxczMhYdARQGKwEiJj0BNDY7ATIWHQEUBisBIiY9ATQ2MzsBMhYdARQGKwEiJj0BNDZJHisrHgLcHiskLkBALiQrHv0kJQKSDxZtDxYWD20WD/1uDxYWD0lJDxYWD0kPFhbqSRAVFRBJDxUVD9xJDxUVD0kPFhYDJSsf/kofKysfSUAtSi1ASR8rShUPbhUPSg8Vbg8VFQ8Bbg8VSRUP3A8VFQ/cDxUVD9wPFRUP3A8VFQ/cDxUVD9wPFQAAAAQAkgBJA7cDtwAPACAAMABAAAATMzIWHQEUBisBIiY9ATQ2NzMyFhURFAYrASImNRE0NjMBMzIWFREUBisBIiY1ETQ2BzMyFhURFAYrASImNRE0NrdJDxYWD0kPFhbqSRAVFRBJDxUVDwG3SQ8WFg9JDxUVzEkPFRUPSQ8WFgElFg+SDxYWD5IPFtsVEP6TDxYWDwFtEBUBtxYP/NwPFhYPAyQPFtwVD/23DxYWDwJJDxUAAAMAAAAAA/8EAAAMABkAJgAAExUWBBYSHwEmAgAkJwMVHgMfAS4DJwEUBiMiJjU0NjMyFhUAtAE87YwEkgSj/uz+j9IBaLeKUwOSA2qx7IYBJFY9PFZWPD1WA/+SBIzt/sSzAdIBcgEVowP+k5MEUoq2aAGH7LFqBP4APFZWPD1WVj0AAAgACgAMA/ED7wALAB0AKQA1AEIATwBiAG4AAAE+ARceAQcOAScuARM4ATE+ARceAQc4ATEOAScuAQc+ARceAQcOAScuASU+ARceAQcOAScuAQM+ARceAQcOAScuATcTPgEXHgEHDgEnLgE3JTgBMT4BFx4BBzgBMQ4BJy4BNwEOAScuATc+ARceAQI2HHAzNCEbHG80MyLfF1wqKhwXFlwqKxvaFFMmJxkVFFMmJxn+2xJMIiMXExJLIyMXsxFEIB8VERFEHyAUECcPPh0cEw8QPR0cEw8BAQ44GRoRDg04GhoRDgK/GWYuLx8ZGWYuLx8DtTQiHBtwNDMiHBtw/esqHBcWXCorGxYXXKknGRUUUycmGRQVU0EjFxMSTCIjFxMSTAEYIBQQEUQgHxUREUQfAT8cEw8PPh0cExAPPR3KGhEODjgaGRENDjga/oguHxkZZS8uHxkZZQAAAAEAJwAnBAAEAABAAAABIg4CBw4BHgEXHgI2Nz4DNTA0NTQmIyIGFRwBMRQOAgcOAS4BJy4CNjc+AzMwMjMyNjU0JiMqATECAE6Re2EeHg4dRzc3hpKXSEh0Uy0gFxcgI0FcODl2c2krLDcXCxcYTGFyPQEBFiAgFgEBBAAtU3RISJeShjc3Rx0OHh5he5FOAQEWICAWAQE9cmFMGBcLFzcsK2lzdjk4XEEjIBcXIAAAAgAAABEEAAPvADoAcgAAAS4BJy4BJy4BJy4BBw4BBw4BBw4BBw4BFx4BFx4BFx4BFx4BNz4BNz4BNz4BNz4BNzoBMzI2NTwBNTEHDgEHDgEHDgEnLgEnLgEnLgEnLgE3PgE3PgE3PgE3PgEXHgEXHgEXHgEXHgEHMRwBFRQWFw4BBwQAARYVFTslJFcvL2QyM2IuLlEjIjUSEhEBARUUEzcjI1AtLF4vL1wqK0wgIDIRCg0EAQIBGydpEjQgIUspKVgrLFYoJ0ceHi4PDw8BARMREDEeHkYmJlEpKE8lJUEcGysODg0BIhkFEAsB7zRmLy9VIyQ3EhMRAQEVFBQ6IyRULS5hMTBfLSxPISE0EREQAQEUExM1IiFOKxo2GycbAQMBrylKHx4wEBAPAQETERIyHx9JKCdUKitSJiZFHB0sDw8NAQERERAvHB1EJCVNKAEDARolAxs0GQAADAAAAAAEAAQAABAAMABQAHAAkAChALIA0gDyARIBMgFDAAABIgYdARQWOwEyNj0BNCYrAQciBg8BDgEVFBYfAR4BMzI2PwE+ATU0Ji8BLgEjMCIxITgBMSIGDwEOARUUFh8BHgEzMjY/AT4BNTQmLwEuASMFOAExIgYPAQ4BFRQWHwEeATMyNj8BPgE1NCYvAS4BIyEiBg8BDgEVFB
```

Comment:
For ttf should be "data:application/octet-stream;" instead of "data:application/font-ttf;"